frontend/public/main.scss

```diff
@@ -0,0 +1,177 @@
+@import "./scss/variables";
+@import "./scss/resets";
+@import "./scss/landingPage";
+@import "./scss/dashboard";
+@import "./scss/basicInfo";
+@import "./scss/team";
+@import "./scss/jira";
+
+.btnCustom {
+    background-color: $color-primary !important;
+    color: white;
+    border-radius: $border-radius !important;
+}
+.inputCustom {
+    border-radius: $border-radius !important;
+    color: black;
+}
+
+.ml1 {
+    margin-left: 0.5rem !important;
+}
+
+@media screen and (max-width: 1450px) {
+    .jiraWid {
+        &__search {
```

Comment:
don't use nested css. it is more costly to interpret when DOM tree structure is made.